server/redis.repository.js

```diff
@@ -0,0 +1,81 @@
+const redis = require("redis");
+const { promisify } = require("util");
+const client = redis.createClient();
+const hgetAsync = promisify(client.hget).bind(client);
```

Comment:
NIT: can we name these variables to something like hGetAsync? This is just to enforce readability

Reply:
The Redis keyword is `hget` and not `hGet`. Since Redis recognizes it as one word I treated it the same way. Let me know what you think.

Reply:
ok that makes sense!